src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -416,7 +416,7 @@ update msg model loggedIn =
                                 value =
                                     String.fromFloat claim.action.verifierReward
                                         ++ " "
-                                        ++ Eos.symbolToSymbolCodeString claim.action.objective.community.symbol
+                                        ++ Eos.symbolToSymbolCodeString loggedIn.selectedCommunity
```

Comment:
hmmm I'm a little concerned about this... but i don't know really why

Reply:
I added the `action.objective.community.symbol` to the `Action` type and reverted changes like this.